### PR TITLE
Setup Mockbee and basic routing

### LINKED ISSUE #1: Create basic mockbee setup and routing

### PR DESCRIPTION
**Pull request by kjais1720**
PR for Issue #1 ( closes #1 )
## Overview
- Setup mockbee
- Setup basic routing to all the pages

### CODE REVIEW

**Review by coldpigli — APPROVED** [review]
Everything looks great.
Liked that you extracted routing into a component

File: src/App.css [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+}
+
+@media (prefers-reduced-motion: no-preference) {
+  .App-logo {
+    animation: App-logo-spin infinite 20s linear;
+  }
+}
+
+.App-header {
+  background-color: #22201c;
```
Comment (rahulcs754) [inline src/App.css:17]: try to use color variable

**Review by rahulcs754 — APPROVED** [review]
The code looks clean and good 
It is basic setup for project

File: src/App.css (hunk 1 above)
Comment (kjais1720) [reply src/App.css:17]: This was pre defined css when created the setup for mockbee, I'll remove it in the next PR

**Pull request merged**